imap/mboxlist.c

```diff
@@ -590,7 +590,7 @@ static int mboxlist_parse_entry(mbentry_t **mbentryptr,
 
     if (name) {
       /* copy name */
-        snprintf(mboxname, MAX_MAILBOX_NAME, "%.*s",
+        snprintf(mboxname, MAX_MAILBOX_NAME+1, "%.*s",
```

Comment:
If this had been `snprintf(mboxname, sizeof(mboxname), ...` in the first place it never would have been a problem, so maybe the fix should be to use sizeof ;)

Reply:
Done!